crates/forge-runner/src/coverage_api.rs

```diff
@@ -12,6 +14,15 @@ pub fn run_coverage(saved_trace_data_paths: &[PathBuf]) -> Result<()> {
         .ok()
         .unwrap_or_else(|| PathBuf::from("cairo-coverage"));
 
+    ensure!(
+        which(coverage.as_os_str()).is_ok(),
+        indoc! {
+            r"The 'cairo-coverage' binary was not found in PATH. It may not have been installed.
+            Please refer to the documentation for installation instructions:
+            https://github.com/software-mansion/cairo-coverage/blob/main/README.md"
```

Comment:
nit:
```suggestion
            https://github.com/software-mansion/cairo-coverage/blob/main/README.md#installation"
```

Reply:
I think `#installation` heading could technically change, but it is very unlikely we ever remove `README.md`. I'd keep it as is. A bit unpractical but should always work.